plugins/modules/packet_device.py

```diff
@@ -423,7 +423,7 @@ def get_hostname_list(module):
         if re.search(r"%\d{0,2}d", hostname_spec):
             hostnames = [hostname_spec % i for i in count_range]
         elif count > 1:
-            hostname_spec = "%s%%02d" % hostname_spec
+            hostname_spec = "%s%%02d" % hostname_spec  # noqa
             hostnames = [hostname_spec % i for i in count_range]
```

Comment:
```suggestion
            hostnames = [f"{hostname_spec}{i:02}" for i in count_range]
```

Reply:
Again, this solves only a part of the problem, line 424 above also deals with `%` and they are coming from the user. I want to find a better solution for this, but not in this PR.